app/models/photo.rb

```diff
@@ -0,0 +1,7 @@
+class Photo < ActiveRecord::Base
+
+  has_attached_file :image, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/images/:style/missing.jpg"
+  validates_attachment_content_type :image, content_type: /\Aimage\/.*\Z/
+  validates_attachment_file_name :image, matches: [/png\Z/, /jpe?g\Z/]
+  validates_presence_of :image, message: "required"
```

Comment:
Rails/Validation: Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.